src/components/VersionSelect.tsx

```diff
@@ -120,7 +120,8 @@ export const VersionSelect = ({
       }
     });
 
-    sortedVersions = result.sort(compareVersions);
+    const validVersions = result.filter(v => validate(v));
```

Comment:
There should probably be a short note in the code here about intent. Something like:

```suggestion
    // remove nonstd versions (eg 0.5.0.pre) to avoid error on sort
    const validVersions = result.filter(v => validate(v));
```

Reply:
Sounds good to me - updated

Reply:
LGTM. Resolving.